src/main/java/space/obminyashka/items_exchange/service/impl/LocationServiceImpl.java

```diff
@@ -123,6 +134,53 @@ public String createParsedLocationsFile(List<RawLocation> creatingData) throws I
         return Files.readString(Path.of(locationInitFilePath).toAbsolutePath(), StandardCharsets.UTF_8);
     }
 
+    @Override
+    public String createParsedAreasFile(List<RawLocation> creatingData) throws IOException {
+        List<Area> areas = mapCreatingDataToAreas(creatingData);
+        String initString = "INSERT INTO area (id, name_ua, name_en) VALUES";
+        try (BufferedWriter writer = Files.newBufferedWriter(Path.of(areaInitFilePath), StandardCharsets.UTF_8)) {
+            writer.append(initString);
+            for (int i = 0; i < areas.size(); i++) {
+                Area area = areas.get(i);
+                writer.append(areaToFormatterString(area));
+                if (i < areas.size() - 1) writer.append(",");
+            }
```

Comment:
You asked to work with streams and ignored that opportunity right here.
StringJoiner also could be used here or .joining() terminal operation of the steam APi